src/test_workflow/dependency_installer.py

```diff
@@ -25,24 +25,26 @@ def __init__(self, root_url, build_manifest, bundle_manifest):
         self.bundle_manifest = bundle_manifest
 
     def download_dist(self, dest):
-        local_path = os.path.join(dest, os.path.basename(self.bundle_manifest.build.location))
+        local_path = os.path.realpath(os.path.join(dest, os.path.basename(self.bundle_manifest.build.location)))
         return self.download_or_copy(self.bundle_manifest.build.location, local_path)
 
+    def __source_dest(self, path, category, dest):
+        source = "/".join([self.root_url, category, self.build_manifest.build.filename, path])
```

Comment:
Seems like this should be coming from the distribution.find_build_root rather than trying to assemble this path again

Reply:
I've tried to use this, but it requires refactoring passing in of root_url here. I'll leave it alone for now. https://github.com/opensearch-project/opensearch-build/issues/1309